common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -696,12 +698,28 @@ function (VideoPlayer, i18n) {
 
     function getDuration() {
         try {
-            return this.metadata[this.youtubeId()].duration;
+            return convertISO8601ToSeconds(this.metadata[this.youtubeId()].duration);
```

Comment:
Might be better to use [moment.js](http://momentjs.com/) for this sort of thing. It's already in our project.